src/Components/Facility/Consultations/components/SharedTable.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { formatDate, formatTime } from "../../../../Utils/utils";
+import { classNames } from "../../../../Utils/utils";
+import { useTranslation } from "react-i18next";
+
+interface SharedSectionTableProps {
+  data: Record<string, any>;
+  rows: Array<{ title?: string; field?: string; subField?: boolean }>;
+  choices?: Record<string, Record<number | string, string>>;
+}
+
+const LogUpdateAnalayseTable: React.FC<SharedSectionTableProps> = ({
+  data,
+  rows,
+  choices = {},
+}) => {
+  const { t } = useTranslation();
+
+  // Helper function to get the display value
+  const getDisplayValue = (value: any, field?: string): string => {
```

Comment:
lets try to avoid any as much as possible, incase if the type can be of multiple types like null, boolean, ..., lets do `null | boolean | string | ...`

Reply:
hey @khavinshankar I made some changes. Will this do?